cypress/support/fragments/quickMarcEditor.js

```diff
@@ -207,34 +214,37 @@ export default {
       cy.get(`input[name="${name}"]`).click();
       cy.get(`input[name="${name}"]`).type(`{backspace}{backspace}${tag008HoldingsBytesProperties.specRet0.newValue}`);
     });
-    pressSaveAndClose();
-    HoldingsRecordView.waitLoading();
+    this.pressSaveAndClose();
     return tag008HoldingsBytesProperties.getAllProperties().map(property => property.newValue).join('');
-  },
-  clearTag008:() => {
+  }
+
+  static clearTag008() {
     tag008HoldingsBytesProperties.getAllProperties().forEach(byteProperty => {
       cy.do(QuickMarcEditorRow({ tagValue: '008' }).find(byteProperty.interactor).fillIn(''));
     });
-    pressSaveAndClose();
-    HoldingsRecordView.waitLoading();
+    this.pressSaveAndClose();
```

Comment:
Same comment here

Reply:
we can, see https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Classes/static